static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -262,6 +269,8 @@ export class DeprecatedLine extends Component<Props, State> {
 
   renderDefaultLine() {
     const {isHoverPreviewed, debugFrames, data} = this.props;
+    const organization = this.props.organization;
+    const anrCulprit = analyzeFrameForRootCause(data, getCurrentThread(this.props.event));
```

Comment:
thinking if we should actually special-case this into a separate `renderAndroidLine()` function, because for now this is gonna be executed for any non-native line

Reply:
only doing this for ANRs now https://github.com/getsentry/sentry/pull/49673/commits/4aa8dad23b40f8f34f8f8de0457b649e50702ece